layouts/shortcodes/conditional-text.html

```diff
@@ -1,36 +1,36 @@
 <!-- Get the current buildcondition from the config and lowercase it -->
-{{ $condition := lower $.Site.Params.buildCondition }}
+{{- $condition := lower $.Site.Params.buildCondition -}}
```

Comment:
Thanks for cleaning up the whitespace issues in this shortcode @fekete-robert.

For next time, it is best to only add the `-` marker where it is needed. As a general rule of thumb, I favor the end tag `-}}`. More rarely do we need `-` on an opening tag, and certainly not in the case where two directives follow each other like this:

```nocode
{{ if ... -}}
  {{ $var := val -}}
  {{ $var := val2 -}}
...
```

/cc @LisaFC